doc/rst/language/spec/tuples.rst

```diff
@@ -872,47 +834,54 @@ For example, in this code:
 
       (0, 0, (x = 0))
 
-Initialization of the tuple variable ``tup`` will make a copy of the
-array ``a``, the record ``r``, and the integer ``i``. Because ``tup`` stores
-a copy of these three variables, changes made to them are not visible
-in ``tup`` when it is written to standard output.
+   Initialization of the tuple variable ``tup`` will make a copy of the
+   array ``a``, the record ``r``, and the integer ``i``. Because ``tup`` stores
+   a copy of these three variables, changes made to them are not visible
+   in ``tup`` when it is written to standard output.
+   Copy-initialization of the tuple's record component would be changed
+   to move-initialization if the record ``r`` were not used later.
 
 .. _Tuple_Argument_Intents:
 
 Tuple Argument Intents
-~~~~~~~~~~~~~~~~~~~~~~
+----------------------
 
-A tuple argument to a function may be either a referential tuple or a value
+A formal argument of a tuple type may be either a referential tuple or a value
 tuple depending on its argument intent.
 
-If the tuple argument has the default argument intent, then it is a 
-referential tuple and some of its elements may be captured by ``ref``
-depending on their default argument intent.
+- If the tuple argument has the default argument intent, then it is a 
+  referential tuple and some of its elements may be captured by ``ref``
+  depending on their default argument intent.
+  If a value tuple (such as a tuple variable) is passed to it, the value tuple
+  will be implicitly converted into a referential tuple. The resulting
+  referential tuple may refer to components from the original value tuple.
 
-A tuple argument declared with ``const`` intent will work similarly to one
-with a default intent, except that all the elements of the tuple are
-considered to be ``const`` and cannot be modified.
+..
```

Comment:
What do single `..` do? Are they page breaks?

Reply:
More like paragraph breaks. They allow different kinds of blocks to coexist next to each other.

In this case they cause empty lines between the bullets. Without those lines the bullets look too condensed.